js/index.js

```diff
@@ -1,5 +1,97 @@
+
+let btns = document.querySelectorAll('button');
+for(btn of btns) {
+    if (btn.innerHTML === 'Cº') {
+        btn.addEventListener('click', function () {
+            fetch("https://api.open-meteo.com/v1/forecast?latitude=27.907&longitude=-82.6909&current=temperature_2m,relative_humidity_2m,is_day,precipitation&timezone=America%2FNew_York&forecast_days=1&models=gfs_seamless")
+            .then(response => {
+                if(!response.ok) {
+                    throw new Error("Not Found");
+                }
+                return response.json();
+            })
+            .then(report => {
+                if(report.length === 0) {
+                    throw new Error("No report exist");
+                } else {
+                    console.log(report);
+                } 
+                
+                const units = document.querySelector('.c_unit');
+                units.innerHTML = report.current_units.temperature_2m;
+
+                 //Displaying reports
+                const today = document.querySelector('#today');
+                const todayData = today.querySelector('p');
+                todayData.innerHTML = report.current.temperature_2m + " " + report.current_units.temperature_2m;
+                
+            })
+            .catch(error => {
+                console.log(error);
+            })
+        })
+    } else if (btn.innerHTML === 'F'){
+        btn.addEventListener('click', function () {
+            fetch("https://api.open-meteo.com/v1/forecast?latitude=27.907&longitude=-82.6909&current=temperature_2m,relative_humidity_2m,is_day,precipitation&temperature_unit=fahrenheit&timezone=America%2FNew_York&forecast_days=1&models=gfs_seamless")
+            .then(response => {
+                if(!response.ok) {
+                    throw new Error("Not Found");
+                }
+                return response.json();
+            })
+            .then(report => {
+                if(report.length === 0) {
+                    throw new Error("No report exist");
+                } else {
+                    console.log(report);
+                } 
+            
+                const units = document.querySelector('.f_unit');
+                units.innerHTML = report.current_units.temperature_2m;
+
+                //Displaying reports
+                const today = document.querySelector('#today');
+                const todayData = today.querySelector('p');
+                todayData.innerHTML = report.current.temperature_2m + " " + report.current_units.temperature_2m;
+            })
+            .catch(error => {
+                console.log(error);
+            })
+        })
+    }
+    
+}
+
+// Displaying city, default current temp and the hourly temperature
+
+fetch("https://api.open-meteo.com/v1/forecast?latitude=27.907&longitude=-82.6909&current=temperature_2m,relative_humidity_2m,is_day,precipitation&temperature_unit=fahrenheit&timezone=America%2FNew_York&forecast_days=1&models=gfs_seamless")
```

Comment:
At this point, it doesn't look like you're using the relative humidity and precipitation. I'd recommend removing those from your API call unless you were planning on using them.